src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -422,15 +420,15 @@ public TextAssessmentResource(AuthorizationCheckService authCheckService, TextAs
     }
 
     /**
-     * With given feedbackConflictId, finds the conflict and sets it as solved.
+     * POST exercise/:exerciseId/feedbackConflict/:feedbackConflictId/solve-feedback-conflict : With given feedbackConflictId, finds the conflict and sets it as solved.
      * Checks; if the feedback conflict is present, if the user is the assessor of one of the feedback or
      * if the user is at least the instructor for the exercise.
      *
      * @param exerciseId - exercise id to check access rights.
      * @param feedbackConflictId - feedback conflict id to set the conflict as solved
      * @return - solved feedback conflict
      */
-    @GetMapping("/exercise/{exerciseId}/feedbackConflict/{feedbackConflictId}/solve-feedback-conflict")
+    @PostMapping("/exercise/{exerciseId}/feedbackConflict/{feedbackConflictId}/solve-feedback-conflict")
```

Comment:
I would argue this should be PUT and could be shorter:

Reply:
```suggestion
    @PutMapping("/exercise/{exerciseId}/feedbackConflict/{feedbackConflictId}/solve")
```

Reply:
Shortening the paths makes sense, however i would also stick with POST here. PUT should be sent with a body, which wasnt the case before and isnt needed here. Also same reason as above, POST should be used to trigger remote methods and doesnt need to be sent with a body.

Reply:
where do see this information that `POST should be used to trigger remote methods`?

Reply:
![image](https://user-images.githubusercontent.com/80530171/117462487-066cc180-af4f-11eb-96bb-f75be09fdf45.png) copied out of https://opac.ub.tum.de/TouchPoint/perma.do?q=+0%3D%22ZDB-30-ORH-047582669%22+IN+%5B2%5D&v=tum&l=de (Chapter 3: "Interaction Design with HTTP"); One more reason would be that PUT is an idempotent operation while POST isnt, and since triggering this specific method isnt idempotent, i think that POST should be used here.